OpenCL_Ext.txt

```diff
@@ -48,6 +48,7 @@ include::ext/cl_khr_int64_atomics.asciidoc[]
 include::ext/cl_khr_select_fprounding_mode.asciidoc[]
 include::ext/cl_khr_gl_sharing__context.asciidoc[]
 include::ext/cl_khr_gl_sharing__memobjs.asciidoc[]
+include::ext/cl_khr_extended_versioning.asciidoc[]
```

Comment:
I'd recommend adding this extension to the end of the doc, after `cl_khr_device_uuid`, to keep reasonably consistent chapter numbers.

Reply:
Good point. I can't think of a reason why I added it there (probably came out of a conflict resolution).